our-initiatives/tutorials/regression.md

```diff
@@ -0,0 +1,11 @@
+---
+sidebar_position: 4
+---
+
+# 2: Regression
+
+**Date: 25th October 2023**
+
+💡 **Linear regression** in simple terms is finding the **line of best fit** given a set of data points that loosly follow a linear pattern. One way of doing this is by **minimizes the loss** (difference between actual and computed value), making it useful for **predicting numerical outcomes** based on input features. **Polynomial regression** extends linear regression by allowing the model to capture **nonlinear relationships** between variables, using polynomial functions to fit the data more flexibly. 💡
```

Comment:
Typo, proposed fix below. Also, I like the explanation that in simple terms it is finding the line of the best fit but this is the case just with 2d feature space. For 3 features that's gonna be a place, for >3 a hyperplane, may be worth specifying that line is just for 2d? Just a note, decision is 100% up to you!
```suggestion
💡 **Linear regression** in simple terms is finding the **line of best fit** given a set of data points that loosely follow a linear pattern. One way of doing this is by **minimising the loss** (difference between actual and computed value), making it useful for **predicting numerical outcomes** based on input features. **Polynomial regression** extends linear regression by allowing the model to capture **nonlinear relationships** between variables, using polynomial functions to fit the data more flexibly. 💡
```